<launch>
    <node pkg="decision_maker" type="decision_maker_node" name="decision_maker_node" output="screen">
	<param name="target_velocity" type="double" value="0.4"/>

        <!-- controller param -->
        <param name="accel" type="double" value="0.2"/>
        <param name="steer_offset" type="double" value="0.05"/>
        <param name="p_gain" type="double" value="0.5"/>
        <param name="i_gain" type="double" value="0.0005"/>
        <param name="d_gain" type="double" value="0.5"/>
        <param name="menual_start" type="bool" value="true"/>
    </node>
    <node pkg="joy" type="joy_node" name="joy_node"/>
</launch>
